### ISSUE TITLE
Keyboard navigation doesn't work

### ISSUE THREAD

**Issue by dionrhys** [lrsjng/h5ai#366]
I'm not able to navigate a h5ai page using the keyboard. These are just typical page navigation keys: Page Down, Page Up, Home, End, Arrow Keys, and Space.

The search box steals focus when any of these keys are pressed, even though many of them don't have any use in the search box. Would it be possible for key presses to only focus the search box if they're actual text entry keys?

Additionally when I press an actual text character like 'c' while the search box is *not focused*, the search box gets focused but the character is not inputted. This results in a weird scenario where you need to press the first key of your search twice in order to get it in the search box.

The browser I used to test this behaviour is Firefox 33.

**Comment by lrsjng** [comment]
This is fixed in the current versions due to a redesign. Filter field now needs to have the focus, doesn't steal all keyboard inputs.
